pkg/reporter/reporter.go

```diff
@@ -238,6 +238,13 @@ func buildPrettyViolationsTable(violations []report.Violation) string {
 		}
 
 		table.Append([]string{yellow("Rule:"), violation.Title})
+
+		// if there is no support for color, then we show the level in addition
+		// so that the level of the violation is still clear
+		if color.NoColor {
+			table.Append([]string{yellow("Level:"), violation.Level})
```

Comment:
Feels a little odd to use `yellow()` here, doesn't it? 🤔

Reply:
😬 hmm indeed!

Reply:
Have updated this now